tests/unit-tests/src/test/java/org/apache/activemq/artemis/tests/unit/core/journal/impl/JournaHistorylBackupTest.java

```diff
@@ -125,11 +126,11 @@ public void removeBKPExtension() throws Throwable {
       String withBKP = withoutBkp + ".bkp";
       // I was actually born at 4:30 :) but I need all numbers lower than 2 digits on the test
       assertEquals(withoutBkp, journal.removeBackupExtension(withBKP));
-      assertEquals(withoutBkp, journal.removeBackupExtension(withoutBkp)); // it should be possible to do it
-
-      String withoutBKP = "jrn-1.data";
+      assertEquals(withoutBkp, journal.removeBackupExtension(withoutBkp));
    }
 
+   // it should be possible to do it
```

Comment:
Doesnt seem like this should have moved here

Reply:
Fixed.